lib/pluginmanager/gemfile.rb

```diff
@@ -15,21 +15,24 @@ def initialize(io)
       @gemset = nil
     end
 
-    def load
+    def load(with_backup = true)
```

Comment:
This is used when we add a plugin, remove a plugin or update a plugin.
By default it will create a a backup of the gemfile. In the context of the pack we don't need that since that class won't write anything to disk.